### PR TITLE
improved AWS client collection naming

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/awsconfig/v8/resource/lifecycle/current.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 			},
 		}
 
-		o, err := r.clients.AutoScaling.DescribeAutoScalingGroups(i)
+		o, err := r.aws.AutoScaling.DescribeAutoScalingGroups(i)
```
Comment (xh3b4sd) [inline service/awsconfig/v8/resource/lifecycle/current.go]: I think this reads nicer and is shorter.

File: service/awsconfig/v8/drainer_resource_set.go [hunk 2]
```diff
@@ -18,8 +18,8 @@ import (
 )
 
 type DrainerResourceSetConfig struct {
-	GuestAWSClients aws.Clients
-	Logger          micrologger.Logger
+	AWS    aws.Clients
```
Comment (xh3b4sd) [inline service/awsconfig/v8/drainer_resource_set.go]: This is more consistent now down the stack.

File: service/awsconfig/drainer_framework.go [hunk 3]
```diff
@@ -150,8 +150,8 @@ func newDrainerResourceRouter(config DrainerFrameworkConfig) (*framework.Resourc
 	var v8ResourceSet *framework.ResourceSet
 	{
 		c := v8.DrainerResourceSetConfig{
-			GuestAWSClients: awsClients,
-			Logger:          config.Logger,
+			AWS:    awsClients,
```
Comment (kopiczko) [inline service/awsconfig/drainer_framework.go]: When I was hacking on aws-operator I noticed distinguishing between guest and host AWS clients is hard, therefore I think it's good to add `Guest` or `Host` to AWS client names.

Comment (rossf7) [reply service/awsconfig/drainer_framework.go]: I agree this is hard currently and knowing if the client is guest or host is very useful.

Comment (xh3b4sd) [reply service/awsconfig/drainer_framework.go]: I found it odd to differentiate something where there is no potential of collision. This is for the lifecycle resource where there is no host client magic. Do you want to be explicit even though there is only the guest client?

Comment (rossf7) [reply service/awsconfig/drainer_framework.go]: That's fair its only the CloudFormation resource that uses both. For me having guest in the name isn't a problem but if you want it to be shorter I'm fine with it.

Comment (kopiczko) [reply service/awsconfig/drainer_framework.go]: Same.

Comment (xh3b4sd) [reply service/awsconfig/drainer_framework.go]: Ok going with `AWS`. Thanks.

**Pull request merged**